File: pset1/mario.c
#include<cs50.h>
#include<stdio.h>

int main(void)
{
int n,i,j,k;
do
{
    printf("Height: ");
n = get_int();
}
while((n<0) || (n>23));
for (i=1;i<=n;i++)
{
    for(j=n-i;j>0;j--)
    {
    printf(" ");
    }
    for(k=1;k<=i+1;k++)
    {
    printf("#");
    }
    printf("\n");
}
return 0;
}

File: pset6/mario.py
import cs50

def main():
    while True:
        print("Height: ", end = "")
        a = cs50.get_int()
        if a >= 0 and a <= 23:
            break
        
    for i in range(a):
        for j in range(a - i - 1):
            print(" ", end = "")
        for k in range(i+2):
            print("#", end = "")
        print("")
            
if __name__ == "_main_":
    main()